api/routes/activities/index.test.js

```diff
@@ -63,7 +63,20 @@ tap.test('activities GET endpoint', async endpointTest => {
     );
 
     handlerTest.test('sends back a list of activities', async validTest => {
-      const activities = [{ id: 1, name: 'hi' }, { id: 2, name: 'bye' }];
+      const get = sinon.stub();
+      get
+        .withArgs('id')
+        .onFirstCall()
+        .returns(1)
+        .onSecondCall()
+        .returns(2);
+      get
+        .withArgs('name')
+        .onFirstCall()
+        .returns('hi')
+        .onSecondCall()
+        .returns('bye');
+      const activities = [{ get }, { get }];
```

Comment:
Tests updated to reflect using the bookshelf `get` methods instead of returning the raw bookshelf objects.